controller_manager/scripts/spawner

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# Copyright 2021 PAL Robotics S.L.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from spawner import *
```

Comment:
The main code is still in `spawner.py` to be moved later.

This is because python relative imports expect an `.py` extension. Otherwise I'd have to add more magic to make it work.

Reply:
Not a problem!